tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/SymbStateDecoder.scala

```diff
@@ -24,16 +24,21 @@ class SymbStateDecoder(solverContext: SolverContext, rewriter: SymbStateRewriter
   private val protoSeqOps = new ProtoSeqOps(rewriter)
   private val recordOps = new RecordAndVariantOps(rewriter)
 
+  def evalGroundExprForBuilderEx(ex: TBuilderInstruction): TBuilderInstruction =
```

Comment:
do we actually need this one? I don't think SMT models have anything scoped

Reply:
I just wrote 2 versions for every call that got mapped over `TBuilderInstruction ` originally. The overload allows you to just swap between importing the scope-safe or unsafe versions.